draft-ietf-rats-eat.md

```diff
@@ -954,11 +963,20 @@ security state of the entity storing the private key used in a PoP application.
 
 ## The Profile Claim (profile) {#profile-claim}
 
-The profile claim is a text string that simply gives the name of the profile to which the token purports to adhere to.
-It may name an IETF document, some other document or no particular document.
+See {{profiles}} for the detailed description of a profile.
+
+A profile is identified by either a URL or an OID.
+Typically, the URL will reference a document describing the profile.
+An OID is just a unique identifier for the profile.
+It may exist anywhere in the OID tree.
 There is no requirement that the named document be publicly accessible.
+An OID is just a unique identifier for the profile.
+It may exist anywhere in the OID tree.
+The primary purpose of the profile claim is to uniquely identify the profile even if it is a private profile.
 
-See {{profiles}} for a detailed description of a profile.
+The OID is encoded in CBOR according to {{CBOR-OID}} and the URI according to {{RFC8949}}.
+Both are unwrapped and thus not tags.
```

Comment:
```suggestion
Both are unwrapped and thus the outer tag is removed.
```

Reply:
I'm leaving this as is because, there is no possibility of any tag here (no inner tag). Also, contrary to normal English use of the word tag, in CBOR the tag is the whole thing with the value, not just the tag number.